Marlin/Conditionals_LCD.h

```diff
@@ -360,8 +360,12 @@
     #if ENABLED(Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN)
       #undef Z_MIN_ENDSTOP_INVERTING
       #define Z_MIN_ENDSTOP_INVERTING false
+      #undef Z_MIN_PROBE_ENDSTOP_INVERTING
+      #define Z_MIN_PROBE_ENDSTOP_INVERTING false
       #define TEST_BLTOUCH() _TEST_BLTOUCH(Z_MIN)
     #else
+      #undef Z_MIN_PROBE_ENDSTOP_INVERTING
+      #define Z_MIN_PROBE_ENDSTOP_INVERTING false
```

Comment:
I believe so.  @tcm0116 is using a clone, and I'm using a genuine, and it works for both of us.